homeassistant/components/mold_indicator/config_flow.py

```diff
@@ -0,0 +1,115 @@
+"""Config flow for Mold Indicator integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN, SensorDeviceClass
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.const import CONF_NAME
+from homeassistant.core import DOMAIN as HOMEASSISTANT_DOMAIN
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import selector
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.issue_registry import IssueSeverity, async_create_issue
+
+from .const import (
+    CONF_CALIBRATION_FACTOR,
+    CONF_INDOOR_HUMIDITY,
+    CONF_INDOOR_TEMP,
+    CONF_OUTDOOR_TEMP,
+    DEFAULT_NAME,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+ISSUE_PLACEHOLDER = {"url": "/config/integrations/dashboard/add?domain=mold_indicator"}
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_INDOOR_TEMP): selector.EntitySelector(
+            selector.EntitySelectorConfig(
+                domain=SENSOR_DOMAIN,
+                filter=selector.EntityFilterSelectorConfig(
+                    device_class=SensorDeviceClass.TEMPERATURE
+                ),
+            ),
+        ),
+        vol.Required(CONF_OUTDOOR_TEMP): selector.EntitySelector(
+            selector.EntitySelectorConfig(
+                domain=SENSOR_DOMAIN,
+                filter=selector.EntityFilterSelectorConfig(
+                    device_class=SensorDeviceClass.TEMPERATURE
+                ),
+            ),
+        ),
+        vol.Required(CONF_INDOOR_HUMIDITY): selector.EntitySelector(
+            selector.EntitySelectorConfig(
+                domain=SENSOR_DOMAIN,
+                filter=selector.EntityFilterSelectorConfig(
+                    device_class=SensorDeviceClass.HUMIDITY
+                ),
+            ),
+        ),
+        vol.Optional(CONF_CALIBRATION_FACTOR): vol.Coerce(float),
```

Comment:
Instead of asking the user to provide this, I think the config flow should ask for the critical temperature, i.e. the current coldest temperature in the room.

Reply:
I think that would probably be good, maybe as a separate PR in future though? 🤔 I'm not sure at what step (when importing from YAML) it would get the coldest temperature. Or whether it would just reverse the calculation of the calibration factor.

Reply:
I would also put that idea in a future PR. I can imagine going a route where the user can either put a pre-calculated calibration factor or the temperatures needed to calculate the factor.

Reply:
I meant that the user is allowed to input the coldest temperature, but we calculate and store the calibration factor.
Is there any reason why the user would like to calibrate the calibration factor by themselves?